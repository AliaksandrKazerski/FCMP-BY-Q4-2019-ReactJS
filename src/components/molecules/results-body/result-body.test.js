import React from 'react';
import { shallow } from 'enzyme';

import ResultsBody from './index';

jest.mock('../../../../img/netflix2.png', ()=>'mock.png');

describe('ResultsBody', () => {
  it("ResultsBody should render correctly", () => {
    const mockSearchParams = {
      offset: 0,
      limit: 0
    };
    const mockPaginationParams = {
      limit: 0,
    };
    const mockSetActivePage = jest.fn;
    const mockMovies = ['1', '2'];

    const myComponent = shallow(
      <ResultsBody
        searchParams={mockSearchParams}
        paginationParams={mockPaginationParams}
        setActivePage={mockSetActivePage}
        movies={mockMovies}
      />
    );
  
  expect(myComponent).toMatchSnapshot()
  });
});