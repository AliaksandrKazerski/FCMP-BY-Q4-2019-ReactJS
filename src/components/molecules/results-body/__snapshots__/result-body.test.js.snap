// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsBody ResultsBody should render correctly 1`] = `ShallowWrapper {}`;
